### PR TITLE
Test migration sequence from old vault to dss-charter

### PR DESCRIPTION
**Pull request by talbaneth**

### CODE REVIEW

File: src/test/CharterManager-unit.t.sol [hunk 1]
```diff
@@ -403,6 +410,61 @@ contract CharterManagerTest is TestBase {
...
+        // make sure new position is as expected under the manager's ilk
+        (ink, art) = vat.urns(ilk, a.proxy());
+        assertEq(ink, 60 * 1e18);
+        assertEq(art, 41176470588235294117); // 40 * (1.05 / 1.02) * 1e18
+        assertEq(a.dai(), 0);
+        assertEq(a.gems(), 0);
+
+        // due to precision imperfection sin and vice are not zeroed back, but are negligible
+        assert(vat.sin(address(this)) < 1e30);
+        assert(vat.vice() < 1e30);
```
Comment (kmbarry1) [inline src/test/CharterManager-unit.t.sol:465]: The easiest way to "solve" this is by using `suck` and `heal` to transfer the `sin` dust to the Vow.

Comment (kmbarry1) [reply src/test/CharterManager-unit.t.sol:465]: `a != address(Vow) => Vat.sin(a) == 0` is an invariant we should strive to maintain for conceptual reasons, so I am in favor of doing what I suggested above.

Comment (kmbarry1) [reply src/test/CharterManager-unit.t.sol:465]: Actually the even easier solution is to use the Vow as the `w` argument to both `grab` calls--then the `sin` dust just ends up in the right place automatically, no extra work required.

Comment (talbaneth) [reply src/test/CharterManager-unit.t.sol]: Done.

**Review by hexonaut — APPROVED** [review]
LGTM

**Review by kmbarry1 — APPROVED** [review]
LGTM

**Pull request merged**